@uportal/portlet-registry-to-array/src/portlet-registry-to-array.d.ts

```diff
@@ -0,0 +1,36 @@
+export interface Registry {
+  registry: {
+    categories: Category[];
+  };
+}
```

Comment:
why have a property called registry on an interface called Registry? Seems like categories should just be a child of Registry

Reply:
:man_shrugging: that's what the JSON contains.
Would there be a better name for this interface?
`Wrapper`, `RegistryWrapper`, `PortallyJSON`, etc

Reply:
if its only because of the JSON response, perhaps add a `RegistryResponse` ? and then only use it to type the response, but return the `Registry` from the service call. Point being that you can do a transform on the response to make it more friendly for the UI to work with.

Reply:
Updated